packages/auth-password/index.js

```diff
@@ -74,7 +74,8 @@ class PasswordAuthStrategy {
       // TODO: This should call `secretFieldInstance.compare()` to ensure it's
       // always consistent.
       // This may still leak if the workfactor for the password field has changed
-      await secretFieldInstance.generateHash('password1234');
+      const hash = await secretFieldInstance.generateHash('password1234');
```

Comment:
@MadeByMike also tried to patch on TODO comment in line 74, if this is good, can remove the comment.